python/tests/integration_tests/test_request_util.py

```diff
@@ -0,0 +1,108 @@
+import pytest
+import json
+from unittest import TestCase
+from unittest.mock import MagicMock
+from requests import HTTPError
+
+from python.utils.request_util import RunRequest
+
+
+class TestRunRequest(TestCase):
```

Comment:
I don't think we need to use the TestCase / unittest module w/ pytest. 

pytest discovers files w/ tests by searching for test_*.py or *_test.py files and then finds test classes that are prefixed with Test like you have here, or functions outside of classes that are prefixed with test

Reply:
Just removed that and moved all the tests to be their own function rather than class methods